### PR TITLE
1.20 -> 1.21 Release Notes mailing list membership

### PR DESCRIPTION
**Pull request by wilsonehusin**
Adding release notes shadows to mailing list

/assign @palnabarun @savitharaghunathan @kikisdeliveryservice @bai 
/cc @ashnehete @melodychn @pmmalinov @soniasingla

### CODE REVIEW

File: groups/sig-release/groups.yaml [hunk 1]
```diff
@@ -343,11 +343,10 @@ groups:
       - sig-release-leads@kubernetes.io
       - antheabjung@gmail.com # 1.21 Enhancements Lead
       - arunmk@gmail.com # 1.21 RT Enhancements Shadow
-      - celeste@cncf.io # 1.20 Release Notes Shadow
+      - ashnehete@gmail.com # 1.20 Release Notes Shadow
```
Comment (bai) [inline groups/sig-release/groups.yaml:346]:
```suggestion
      - ashnehete@gmail.com # 1.21 Release Notes Shadow
```

Comment (wilsonehusin) [reply groups/sig-release/groups.yaml:346]: oop thanks! fixed it

**Review by soniasingla — APPROVED** [review]
Thanks @wilsonehusin! 🔥

**Comment by dims** [comment]
@wilsonehusin needs rebase please

**Comment by palnabarun** [comment]
/lgtm
/approve

Thank you @wilsonehusin :)

**Pull request merged**